src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicHeaderFacts.vb

```diff
@@ -28,7 +28,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             End If
 
             Dim typeStatement = typeBlock.BlockStatement
-            typeDeclaration = typeStatement
+            typeDeclaration = typeBlock
```

Comment:
this it the fix.  this VB helper doesn't match the rest (like "IsBetweenMembers") and returns teh type statement not the type block.